tests/test_flobject.py

```diff
@@ -745,34 +745,9 @@ def test_accessor_methods_on_settings_object_types(load_static_mixer_settings_on
     assert max_refinements.get_attr("max") == 1000000
 
 
-@pytest.mark.fluent_version("==23.1")
+@pytest.mark.fluent_version("==24.1")
```

Comment:
Keeping only one test as As these tests are testing a pyfluent feature. `find_children(setup_cls(), "*gen")` resturns an empty list in 24.2, hence using 24.1.